src/plugins/remote-cache/routes/get-health.ts

```diff
@@ -0,0 +1,14 @@
+import type { Server } from 'http'
```

Comment:
We should use this route https://github.com/ducktors/turborepo-remote-cache/blob/main/src/plugins/remote-cache/routes/get-status.ts and bump a major version instead of adding a new one.